proposal.md

```diff
@@ -38,25 +38,31 @@ are uneditable. Our application will strive to break from these conventions to p
 user can type wherever they choose.
 
 ## Tasks
-* The application shall require users to have accounts.
-  * Email, Display Name, Password
-  * Multi-Factor Authentication
-  * Google/Facebook authentication
-* The application shall allow user input to be displayed to other authorized users in real-time.
-* Users have "conversations" with each other where the conversation is updated in real-time with other's input
-* Notifications will be when a user is typing/has typed within the conversation
-* When a user re-enters a conversation, they'll be shown the changes made to the conversation since the last time they were 
-there
-* A mini-map can be displayed in the corner of the screen that shows an overview of the conversation and where other users are 
-in it 
-* Lock content
-  * Essentially pinning an excerpt so that it can't be edited
-  * Possibly lock position and content
-* Formatting will be stored as XML
-* Input images and animated gifs
-* Highlight excerpt and get history of it
-  * Which user typed what
-* Search through conversations and users
+This messaging application will allow users to create accounts to communicate
+with each other. The accounts can be created using an email account or with
+Google or Facebook authentication. When signed in, users can view and modify their
+"conversations" with other users.
+
+The base functionality of a conversation should mimic a real-time collaboration
+word processor. Any changes being made to the conversation will be reflected in
+real-time to all other users participating in the conversation. Text, images,
+and gifs will be accepted as valid conversation inputs. The application will
+support text formatting. Conversation content can also be locked by users so
```

Comment:
Could use "styling" instead of "formatting" and maybe list examples like, "The application will support text styling options like bold and italic text" to make the sentence a bit less abrupt. Something like that...